src/main/scala/ClapYourHands.scala

```diff
@@ -1,2 +1,130 @@
-object ClapYourHands extends App {
+//All the scala newbie things
+object scalaNewbieProjects extends App{
+  println("Hello World")
+  var hW = "Hello World"
+  def tryThemAllOut={
+    var word = "Hello"
+    var word1 = "Ha"
+    var word2 = "llo"
+    var char1 = "e"
+    var char2 = "a"
+    var num1 = 4
+    var num2 = 6
+    var TorF = true
+    println(this.returnWord(word))
+    println(this.returnAny(char1))
+    println(this.partOfString(word,2))
+    println(this.replaceAndConcatenateString(word1,word2,char1,char2))
+    println(this.sumOrMultiply(num1,num2,TorF))
+    this.repeatMyself(word,num2)
+    this.wordSquare(word,num2)
+  }
+  def returnWord(word:String):String ={return(word)}
```

Comment:
we don't use keyword "return" in scala, the last expression or statement will be the one to be returned therefore you could just have it like this:

def returnWord(word:String):String = word

go through your methods everywhere and please get rid of keyword return